src/components/Proposals/ProposalsList/ProposalsList.tsx

```diff
@@ -90,30 +90,36 @@ export default function ProposalsList({
       )}
       <VStack className="bg-neutral border border-line rounded-lg shadow-newDefault overflow-hidden">
         <div>
-          <InfiniteScroll
-            hasMore={meta.has_next}
-            pageStart={0}
-            loadMore={loadMore}
-            loader={
-              <div key={0}>
-                <HStack
-                  key="loader"
-                  className="gl_loader justify-center py-6 text-sm text-secondary"
-                >
-                  Loading...
-                </HStack>
-              </div>
-            }
-            element="main"
-          >
-            {proposals.map((proposal) => (
-              <Proposal
-                key={`${proposal.id}_${proposal.status}`}
-                proposal={proposal}
-                votableSupply={votableSupply}
-              />
-            ))}
-          </InfiniteScroll>
+          {proposals.length === 0 ? (
+            <HStack className="justify-center py-8 text-secondary">
```

Comment:
Andrei and I are fierce `HStack` / `VStack` haters 😆 
We typically try to change all of these to 
`<div className="flex flex-row justify-center py-8 text-secondary">`

 I won't make you change this in the PR but in the future we'd love to kill off all of the HStacks. It's an unnecessary layer of abstraction